src/main.ts

```diff
@@ -90,6 +89,12 @@ export async function run(argv?: string[], options?: Interfaces.LoadOptions): Pr
 
   try {
     return await config.runCommand(id, argvSlice, cmd)
+  } catch (error) {
+    // WARNING: error instanceof NonExistentFlag does not work
+    // WARNING: typeof error === 'NonExistentFlag' does not work
+    if ((error as Error).message.includes('Nonexistent flag')) {
+      await showHelp(argv, config)
+    }
```

Comment:
Also, I don't think we should be showing the entirety of the help output. Often that'll be way too much information for the user to digest, especially when they're not expecting to see the help output

Instead, I think we should either list out the valid flags in the `NonExistentFlagsError` message or figure out a way to show only the `FLAGS` part of the help output. My vote is for the first option since it'll be much simpler

If we did go with some variation of `showHelp` that only shows the `FLAGS`, we'll also need to ensure that the output goes to stderr instead of stdout since it's technically part of an error message